README.md

```diff
@@ -2,44 +2,59 @@
 
 # Home Office Forms Example Form
 
-In order to provide a starting point for people using the [home office forms toolkit](https://github.com/UKHomeOffice/hof) this app aims to give a simple example of how to use the module. We encourage users to clone this repository in order to provide a starting point for their own forms.
+This is an example of a service built with [Home Office Forms](https://github.com/UKHomeOffice/hof), A.K.A HOF. It intends to highlight many of the features available from HOF, but is not exhaustive.
+
+## Items of Note
+
+This is not the only way to configure your HOF service. This service uses a typical Node.JS, Express architecture, with an entry point named `app.js`, which is where the majority of the set up occurs.
+
+The specifics of a form are configured in `apps/`, this is where views, controllers, translations and the steps and fields are defined for each form journey (spolier: there is only one at the moment).
+
+If you wish to use this as a starting point from which to build your own form service, we encourge you to delete the commit history after cloning this repository.
```

Comment:
'encourage'. Also what about how to delete the commit history? Or a link to somewhere explaining